scripts/us_census/acs5yr/subject_tables/s2201/README.md

```diff
@@ -1,26 +1,27 @@
-# Importing Census ACS5Year Table S2201
```

Comment:
Is this relevant in this PR?

Reply:
Will clean this PR in the next commit! There are a few files that are being tracked that can moved to a different PR